packages/node_modules/@node-red/runtime/lib/api/flows.js

```diff
@@ -255,5 +259,83 @@ var api = module.exports = {
             }
         }
         return sendCredentials;
-    }
+    },
+    /**
+    * Gets running state of runtime flows
+    * @param {Object} opts
+    * @param {User} opts.user - the user calling the api
+    * @param {Object} opts.req - the request to log (optional)
+    * @return {{state:string, started:boolean}} - the current run state of the flows
+    * @memberof @node-red/runtime_flows
+    */
+    getState: async function(opts) {
+        runtime.log.audit({event: "flows.getState"}, opts.req);
+        const result = {
+            state: runtime.flows.started ? "started" : "stopped",
+            started: !!runtime.flows.started,
```

Comment:
Why have two properties that say the same thing - one as a string, one as a boolean?

The result of `GET /flows/state` should mirror what can be sent via `POST /flows/state`.

`rev` is part of the `/flows` endpoint and already made available to the editor. Is there a particular need to include it in this response?

Reply:
When testing multiple editors I found it necessary to know the rev in the response (to avoid unnecessary pop-ups about merging etc)

Reply:
If the `rev` hasn't changed (because changing runtimeState shouldn't touch `rev`) then I'm puzzled why that would happen. The other editors will get notified of a changing in runtime state - that shouldn't trigger anything about merging changes.